src/clj/rems/db/licenses.clj

```diff
@@ -113,18 +115,24 @@
   [attachment-id]
   (db/remove-license-attachment! {:id attachment-id}))
 
+(defn- workflow-in-error [wf]
+  (select-keys wf [:id :title]))
+
+(defn- resource-in-error [resource]
+  (select-keys resource [:id :resid]))
+
 (defn- get-license-usage [id]
   ;; these could be db joins
   (let [resources (->> (db/get-resources-for-license {:id id})
                        (map :resid)
-                       (map #(db/get-resource {:id %}))
+                       (map resource/get-resource)
                        (remove :archived)
-                       (map :id))
+                       (map resource-in-error))
         workflows (->> (db/get-workflows-for-license {:id id})
                        (map :wfid)
-                       (map #(db/get-workflow {:wfid %}))
+                       (map workflow/get-workflow)
                        (remove :archived)
-                       (map :id))]
+                       (map workflow-in-error))]
```

Comment:
The name `{resource,workflow}-in-error` brings to my mind an operation that finds workflows/resources which are in an error state. But here the usage is related to selecting only those keys which the UI will show in an error message. I'd inline the functions `(map #(select-keys % [:id :resid]))` or use very long but descriptive function names.